frontend/src/views/DataTableDemo.vue

```diff
@@ -0,0 +1,105 @@
+<template>
```

Comment:
Have we got a task or TODO to remove this once it's not needed as a reference?

Reply:
I will remove this.